nipype/pipeline/plugins/pbs.py

```diff
@@ -47,18 +47,20 @@ def __init__(self, **kwargs):
         super(PBSPlugin, self).__init__(template, **kwargs)
 
     def _is_pending(self, taskid):
-        result = CommandLine('qstat {}'.format(taskid),
+        result = CommandLine('qstat -f {}'.format(taskid),
                              environ=dict(os.environ),
                              terminal_output='allatonce',
```

Comment:
I would suggest changing `terminal_output` to `'files_split`

Reply:
@oesteban why? & where can I read about file_split, it seems to be not documented yet (http://nipype.readthedocs.io/en/latest/interfaces/generated/interfaces.base/core.html#parameters)?

Reply:
@achetverikov you can read about terminal outputs here
http://nipype.readthedocs.io/en/latest/devel/interface_specs.html#controlling-outputs-to-terminal